Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.xaml

```diff
@@ -132,12 +134,10 @@
                                 <ColumnDefinition Width="Auto" />
                             </Grid.ColumnDefinitions>
 
-                            <TextBlock x:Name="TitleBar"
-                                       Margin="4,0,0,0"
```

Comment:
Doesn't look like there's a default template defined which maintains this padding and alignment for the text within the presenter?

We should make sure the visual behavior is consistent still with this type of change.

Reply:
Thanks for pointing that out, fixed!

Reply:
That's what CRs are for :).

Don't forget the HorizontalAlignment Property though too, called it out above.